app/protocol/flows/blockrelay/handle_relay_invs.go

```diff
@@ -237,6 +237,7 @@ func (flow *handleRelayInvsFlow) processAndRelayBlock(requestQueue *hashesQueueS
 			for _, missingAncestor := range missingParentsError.MissingParentHashes {
 				requestQueue.enqueueIfNotExists(missingAncestor)
 			}
+			log.Infof("Received a block with unknown parents, adding to orphan pool: %s", blockHash)
```

Comment:
I'd prefer for this log to be inside AddOrphan
Also please write "missing parents" instead of "unknown parents"